relayer/workers/parachaincommitmentrelayer/beefy-listener.go

```diff
@@ -55,6 +54,26 @@ func (li *BeefyListener) Start(ctx context.Context, eg *errgroup.Group) error {
 	}
 	li.beefyLightClient = beefyLightClientContract
 
+	// Fetch ParaId
```

Comment:
nice improvement - better than putting it in config as you say as we already have the endpoint in config